media_driver/linux/common/os/i915/include/uapi/i915_drm.h

```diff
@@ -62,6 +62,26 @@ extern "C" {
 #define I915_ERROR_UEVENT		"ERROR"
 #define I915_RESET_UEVENT		"RESET"
 
+/*
+ * i915_user_extension: Base class for defining a chain of extensions
```

Comment:
That's a mix of Virtual Engine, SSEU and maybe some other changes to i915 uAPI. Can we, please, separate these changes from each other? We definitely will merge all that step by step. Something like this:
1. Introduce SSEU into i915 uAPI
2. Patch to enable SSEU support in the driver
3. Introduce VE into i915 uAPI
4. Patch to enable VE support in the driver

After that you can create 2 PRs from step-2 and step-4. I think that it is ok if PR for VE will have SSEU as a prerequisite. Closer to the merge we could reorder patches/PRs depending on which feature will land in i915 first.

Reply:
okay.

Reply:
Done. 1) and 2) are in this PR. 3) and 4) are in PR https://github.com/intel/media-driver/pull/283